projects/hslayers/src/components/save-map/save-map.service.ts

```diff
@@ -415,12 +416,20 @@ export class HsSaveMapService {
           url: encodeURIComponent(definition.url),
           format: definition.format,
         };
+        json.workspace = getWorkspace(layer);
         delete json.features;
       } else {
-        try {
-          json.features = this.getFeaturesJson(src.getFeatures());
-        } catch (ex) {
-          //Do nothing
+        if (layer.get('url')) {
```

Comment:
It's quite a bold assumption that if the layer contains url attribute, then it's automatically WFS.

Reply:
Well its Vector layer without definition that has url property. But I can agree its not particulary save. Used it as it was necessary it store for those layers and also seemd to serve this purpose. 
Do we want to mark the layer with another param then?